### DISCUSSION TITLE
How to share a Prisma schema in 2023?

### DISCUSSION THREAD

**Discussion by TotomInc** [prisma/prisma#21097] [Q&A] [answered]
### Question

I've read multiple outdated resources on how to share a Prisma schema, without a monorepo but with npm packages.

Here's a popular outdated article: https://isidoro-ferreiro.medium.com/share-your-prisma-client-across-projects-44d1c7aca6fd

What's the "recommended" way, without a monorepo? I would like to query my Prisma client from a package so I can reuse it on multiple services I have?

**Comment by ludralph** [comment, marked as answer]
Hi @TotomInc 👋 

Thank you for raising this question. 

This question was discussed previously [here](https://github.com/prisma/prisma/discussions/5526).

There is also an example that was linked that shows how to share Prisma schema types and it's also useful in a microservice setup

- https://github.com/prisma/prisma/discussions/5526#discussioncomment-1697570

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by TotomInc** [reply]
Hey @ludralph, thanks for your answer.

It looks like the discussion you mentioned is a bit outdated, but if there's still not a better option than that, then I guess it should do the work.
